Zhuk.University.Tachka.Web/Pages/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,89 @@
+    <link rel="icon" type="image/png" href="~/generalicon.png" sizes="32x32">
+</head>
+
+<body>
+    <header class="p-3 mb-3 border-bottom">
+        <div class="container">
+            <div class="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start">
+                <a href="/" class="d-flex align-items-center mb-2 mb-lg-0 text-dark text-decoration-none">
+                    <svg class="bi me-2" width="40" height="32" role="img" aria-label="Bootstrap"><use xlink:href="#bootstrap"></use></svg>
+                </a>
+                <ul class="nav col-12 col-lg-auto me-lg-auto mb-2 justify-content-center mb-md-0">
+                    <li><a asp-area="" asp-page="/Index" class="nav-link px-2 text-gray">Tachka</a></li>
+                    <li><a class="nav-link px-2 link-secondary" asp-area="" asp-page="/CarList">Cars</a></li>
+                    <li><a href="#" class="nav-link px-2 link-secondary">Suggestions</a></li>
+                    <li><a class="nav-link px-2 link-secondary" href="/swagger">Swagger UI</a></li>
+                    <li><a asp-area="" asp-page="/About" class="nav-link px-2 link-secondary">About</a></li>
+                </ul>
+
+                <form class="col-12 col-lg-auto mb-3 mb-lg-0 me-lg-3" role="search">
+                    <input type="search" class="form-control" placeholder="Search..." aria-label="Search">
+                </form>
+                
+                <a class="nav-link px-2 link-secondary text-gray col-12 col-lg-auto mb-3 mb-lg-0 me-lg-3 " asp-area="Identity" title="Manage">@User.Identity?.Name</a>
+                
+                
+                <div class="dropdown text-end">
+                    <a href="#" class="d-block link-dark text-decoration-none dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false">
+                        <img src="https://github.com/mdo.png" alt="mdo" width="32" height="32" class="rounded-circle">
+                    </a>
+                    <ul class="dropdown-menu text-small">
+                        <li><a class="dropdown-item" asp-area="" asp-page="/CreateCar">Create Car</a></li>
+                        <li><a class="dropdown-item" asp-area="Identity" asp-page="/Account/Manage/Index" title="Manage">Settings</a></li>
+                        <li><a class="dropdown-item" href="#">Profile</a></li>
+                        <li><i class="dropdown-item  bi bi-brightness-high-fill" id="toggleDark"></i></li>
+                        <li>
+                            
+                        </li>
+                        <partial name="_LoginPartial" />
+                        <li><hr class="dropdown-divider"></li>
+                        <form class="dropdown-item" asp-area="Identity" asp-page="/Account/Logout" asp-route-returnUrl="@Url.Page("/", new { area = "" })" method="post">
+                            <button type="submit" class="nav-link btn btn-link text-dark">Logout</button>
```

Comment:
```suggestion
                            <button class="nav-link btn btn-link text-dark" type="submit">Logout</button>
```

By doing so, it allows for easier modification of the button's styles in the future)